R-package/src/install.libs.R

```diff
@@ -42,17 +42,23 @@ if (!use_precompile) {
   setwd(build_dir)
   
   # Prepare installation steps
-  cmake_cmd <- "cmake "
-  build_cmd <- "make _lightgbm -j4"
+  cmake_base <- "cmake "
+  build_cmd <- "make _lightgbm -j"
   lib_folder <- file.path(R_PACKAGE_SOURCE, "src", fsep = "/")
   
   # Check if Windows installation (for gcc vs Visual Studio)
   if (WINDOWS) {
     if (use_mingw) {
-      cmake_cmd <- paste0(cmake_cmd, " -G \"MinGW Makefiles\" ")
-      build_cmd <- "mingw32-make.exe _lightgbm -j4"
+      cmake_cmd <- paste0(cmake_base, " -G \"MinGW Makefiles\" ")
+      build_cmd <- "mingw32-make.exe _lightgbm -j"
+      system(paste0(cmake_cmd, " ..")) # Must build twice for Windows due sh.exe in Rtools
```

Comment:
Double cmake for VS does not cause issues.

This is the message printed when using twice cmake with VS: `CMake does not need to re-run because C:/tmp/RtmpygCFH6/devtools19ad439764e21/Laurae2-LightGBM-ed00f7b/R-package/src/build/CMakeFiles/generate.stamp is up-to-date.`